go/kbfs/libkbfs/settings_db.go

```diff
@@ -0,0 +1,142 @@
+package libkbfs
+
+import (
+	"os"
+	"path"
+	"strconv"
+
+	"github.com/keybase/client/go/kbfs/idutil"
+	"github.com/keybase/client/go/protocol/keybase1"
+	"github.com/pkg/errors"
+	"github.com/syndtr/goleveldb/leveldb/opt"
+	"github.com/syndtr/goleveldb/leveldb/storage"
+	"golang.org/x/net/context"
+)
+
+const (
+	// Where in config.StorageRoot() we store settings information.
+	settingsDBDir           = "kbfs_settings"
+	settingsDBVersionString = "v1"
+	settingsDBName          = "kbfsSettings.leveldb"
+
+	// Settings keys
+	notificationThresholdKey = "notificationThreshold"
```

Comment:
maybe a slightly more descriptive name would be good here, e.g. `spaceAvailableNotificationThreshold` like below, or something? no strong feelings though.